react/components/CostCentersAutocomplete.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import { useQuery } from 'react-apollo'
+import { AutocompleteInput } from 'vtex.styleguide'
+import { useIntl } from 'react-intl'
+
+import { messages } from './customers-admin'
+import GET_COST from '../queries/costCentersByOrg.gql'
```

Comment:
We don't refer to "cost centers" as "cost" nowhere else in the code. Can we rename this to `GET_COST_CENTER`?

Reply:
yes, I will rename

Reply:
`GET_COST_CENTER_BY_ORG`